src/webgpu/api/validation/queue/copyImageBitmapToTexture.spec.ts

```diff
@@ -210,68 +201,50 @@ g.test('source_imageBitmap_state')
     t.runTest(
       { imageBitmap },
       { texture: dstTexture },
-      noopCopy ? { width: 0, height: 0, depth: 0 } : { width: 1, height: 1, depth: 1 },
-      !closed,
+      copySize,
+      true, // No validation errors.
       closed ? 'InvalidStateError' : ''
     );
   });
 
-g.test('destination_texture_state')
+g.test('destination_texture,state')
   .params(
     params()
-      .combine(poptions('destroyed', [true, false]))
-      .combine(poptions('noopCopy', [true, false]))
-  )
-  .fn(async t => {
-    const { destroyed, noopCopy } = t.params;
-    const imageBitmap = await createImageBitmap(t.getImageData(1, 1));
-    const dstTexture = t.device.createTexture({
-      size: { width: 1, height: 1, depth: 1 },
-      format: 'bgra8unorm',
-      usage: GPUTextureUsage.COPY_DST,
-    });
-
-    if (destroyed) dstTexture.destroy();
-
-    t.runTest(
-      { imageBitmap },
-      { texture: dstTexture },
-      noopCopy ? { width: 0, height: 0, depth: 0 } : { width: 1, height: 1, depth: 1 },
-      !destroyed
-    );
-  });
-
-g.test('error_destination_texture')
-  .params(
-    params()
-      .combine(poptions('isErrDstTexture', [true, false]))
-      .combine(poptions('noopCopy', [true, false]))
+      .combine(poptions('state', ['valid', 'invalid', 'destroyed'] as const))
+      .combine(
+        poptions('copySize', [
+          { width: 0, height: 0, depth: 0 },
+          { width: 1, height: 1, depth: 1 },
+        ])
+      )
   )
   .fn(async t => {
-    const { isErrDstTexture, noopCopy } = t.params;
+    const { state, copySize } = t.params;
     const imageBitmap = await createImageBitmap(t.getImageData(1, 1));
-    const correctTexture = t.device.createTexture({
-      size: { width: 1, height: 1, depth: 1 },
+    const descriptor: GPUTextureDescriptor = {
+      size:
+        state === 'invalid' ? { width: 0, height: 0, depth: 0 } : { width: 1, height: 1, depth: 1 },
```

Comment:
With the suggested change inside createTextureWithState, you can change this back to just `size: { width: 1, height: 1, depth: 1 }`.